beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/aws/S3ClientTest.java

```diff
@@ -207,6 +207,21 @@ void deleteObjectsInDirectory() {
     assertThat(amazonS3.doesObjectExist(bucket, key2)).isFalse();
   }
 
+  @Test
+  void deleteThousandObjectsInDirectory() {
```

Comment:
Could you add in this test an assertion where if you try to delete the same quantity of objects using the s3 service provided by the aws library the amount of deleted objects are 1000? (instead of 1234)

Reply:
Changed the test to work with 1000 files instead of 1234

Reply:
if you only have 1000 objects, then the loop will only be executed once. you should add a test where the execution is repeated for at least 2 runs. (at least 1001 object i think)

Reply:
Merged the tests and changed the it to be parameterized